packages/loader/container-loader/src/container.ts

```diff
@@ -1539,7 +1569,7 @@ export class Container
 
 		// Start websocket connection as soon as possible. Note that there is no op handler attached yet, but the
 		// DeltaManager is resilient to this and will wait to start processing ops until after it is attached.
-		if (loadMode.deltaConnection === undefined && !pendingLocalState) {
+		if (loadMode.deltaConnection === undefined) {
```

Comment:
Sorry didn't get why we removed the check for pendingLocalState? Can you please explain.

Reply:
It's no longer needed. It was running into problems before (I mentioned it in description) with immediate connections. So Wes added this check here and in handleDeltaConnectionArg() to special case it. With my changes, no special casing is required, as connections are not observable by runtime through load, so we can simplify code.